.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/openjdk:8-jdk
+
+    working_directory: ~/repo
+
+    environment:
+      JVM_OPTS: -Xmx3200m
+      TERM: dumb
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            - sbt-scala-cache-{{ checksum "build.sbt" }}
```

Comment:
clever. did you make this up yourself? :)

Reply:
Of course not :)